src/db/migrations/20230721030154-save-proofs.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    const transaction = await queryInterface.sequelize.transaction();
+    try {
+      await queryInterface.addColumn(
+        'Games',
+        'gameProof',
+        { type: Sequelize.JSONB, allowNull: true, defaultValue: null },
+        { transaction }
+      );
+      await queryInterface.addColumn(
+        'GamePhases',
+        'turnProof',
```

Comment:
What will the scope of `turnProof` be? Will it include all phases of both player turns for e.g. Turn 1? Or will it just be Player 1's Turn 1 phases?

Reply:
A turn proof is the sum of all phase changes for a single player.

Actions -> Phase
Phases -> Turn
Turns -> Game